R/rules-line-breaks.R

```diff
@@ -447,3 +447,28 @@ remove_empty_lines_after_opening_and_before_closing_braces <- function(pd) {
 
   pd
 }
+
+
+#' Reduce multiple blank lines to a maximum number of allowed blank lines
+#' @param pd_flat A flat parse table.
+#' @param allowed_blank_lines The maximum number of allowed blank lines between code elements. Default is `2L`.
+#' @keywords internal
+reduce_extra_blank_lines_between_scopes <- function(pd_flat, allowed_blank_lines = 2L) {
```

Comment:
Style guide recommends one line breaks, but only for function calls. But that itself felt quite aggressive, given the huge numbers of changes in test outputs.

`2L` nicely aligns with Python and feels "natural" as well.